### DISCUSSION TITLE
How to deploy `workers` by setting worker group to 'group-01' instead of 'default'?

### DISCUSSION THREAD

**Discussion by archongum** [apache/dolphinscheduler#16932] [Q&A] [answered]
According to docs - [Docker Quick Start](https://dolphinscheduler.apache.org/en-us/docs/3.1.5/guide/start/docker).

How to deploy `workers` by setting worker group to 'group-01' instead of 'default'? 
Because I'm going to deploy `workers` in Kubernetes `Deployment` stateless type.

```sh
docker run -d --name dolphinscheduler-worker \
    -e DATABASE="postgresql" \
    -e SPRING_DATASOURCE_URL="jdbc:postgresql://localhost:5432/dolphinscheduler" \
    -e SPRING_DATASOURCE_USERNAME="<USER>" \
    -e SPRING_DATASOURCE_PASSWORD="<PASSWORD>" \
    -e REGISTRY_ZOOKEEPER_CONNECT_STRING="localhost:2181" \
    --net host \
    -d apache/dolphinscheduler-worker:"${DOLPHINSCHEDULER_VERSION}"
```

**Comment by Gallardot** [comment, marked as answer]
https://github.com/apache/dolphinscheduler/pull/16875  supports this feature, which will be released in 3.3.0
